setup.py

```diff
@@ -82,7 +82,7 @@ def build_extension(self, ext):
             env.get('CXXFLAGS', ''),
             self.distribution.get_version())
 
-        env['CXX'] = 'g++'
+        env['CXX'] = 'g++-5'
```

Comment:
Why are we doing this?

Reply:
this was to follow the guidance in https://github.com/tensorflow/custom-op to use the same version of gcc. the TF image referenced in that doc uses gcc 5.4, and g++-5 installs gcc 5.5, but it's as close as @nadiaya was able to get when testing locally.

Reply:
why shouldn't we do this? 
This is third release when we were getting errors because of the local compiler being different.
Also we have g++-5 defined everywhere in the buildspecs.